packages/fx-core/src/question/constants.ts

```diff
@@ -44,6 +44,7 @@ export enum QuestionNames {
   FromExistingApi = "from-existing-api", // group name for creating an App from existing api
   ApiSpecLocation = "openapi-spec-location",
   ApiOperation = "api-operation",
+  ApiPluginManifestPath = "api-plugin-manifest-path",
```

Comment:
might be better to rename or add a comment indicating that this question is for internal integration and won't for UI.

Reply:
Updated.